doc/source/examples/extended_examples/executable/executable.rst

```diff
@@ -101,6 +101,64 @@ This code shows how you can input other arguments:
    The first natural frequency is 344.28 Hz.
 
 
+Convert the app to an executable
+================================
+
+Using the Python library `PyInstaller <https://pyinstaller.org>`, it is possible to convert the app to an executable by taking a few precautions.
+
+.. code:: console
+
+   pyi-makespec cli_rotor.py
```

Comment:
Why these lines? Please explain.

Reply:
I am also assuming you are executing this at the root of the repo.

Reply:
[I have specified to do it at the root of the project](https://github.com/ansys/pymapdl/pull/2100/files#diff-32d8dfd777a43e2e0eb0d74e81e2f7a550c314abd47a3476905733c8c92803e9R111)